src/kcmd/conclusion.go

```diff
@@ -1,6 +1,6 @@
 package kcmd
 
-func GrubConclusion(grubFile string) []string {
+func GrubConclusion(grubFile string, diff []string) []string {
```

Comment:
This should receive the diff (old, new) without formatting and do the formatting internally.